go.mod

```diff
@@ -15,11 +15,11 @@ require (
 	github.com/Psiphon-Labs/utls v0.0.0-20181219022742-11a4cc033322 // indirect
 	github.com/Yawning/chacha20 v0.0.0-20170904085104-e3b1f968fc63 // indirect
 	github.com/agl/ed25519 v0.0.0-20170116200512-5312a6153412 // indirect
-	github.com/apex/log v1.1.0
```

Comment:
I've updated all direct dependencies except Psiphon (as documented in the relevant commit).